conda_lock/conda_solver.py

```diff
@@ -577,6 +578,7 @@ def fake_conda_environment(
                 "build_number": build_number,
                 "version": dep.version,
                 "subdir": path.parent.name,
+                "fn": file_name,
```

Comment:
```suggestion
                "fn": path.name,
```

Reply:
Minor nit, I don't think we gain anything by introducing an extra variable here.